lcss.go

```diff
@@ -20,12 +20,6 @@ import "bytes"
 // of the suffix array sequences.
 func LongestCommonSubstring(strs ...[]byte) []byte {
 	strslen := len(strs)
-	if strslen == 0 {
-		return []byte{}
-	}
```

Comment:
The first one - yes. The second one avoids calling `Qsufsort()` on a single string for no benefit.